codebase/src/java/disco/org/openlvc/disco/connection/rpr/mappers/TransmitterMapper.java

```diff
@@ -403,7 +404,25 @@ private void deserializeFromHla( RadioTransmitter hlaObject, AttributeHandleValu
 		hlaDecode( hlaObject.getRfModulationSystemType(), rfModulationSystemType, map );
 
 		// RFModulationType
-		hlaDecode( hlaObject.getRfModulationType(), rfModulationType, map );
+		try
```

Comment:
An alternative which is much more hacky, but does leave Exceptions only being thrown in purely exceptional circumstances could be to inspect the bytes of the rfModulationType value before decoding. 

- If it's a zero value (Other) then don't bother decoding, and set it to the `Unmodulated` alternative.
- Otherwise call `hlaDecode` as per-usual

Would look like:

```
byte[] modulationTypeRaw = map.get( rfModulationType.getHandle() );
if( modulationTypeRaw.length == 2 && modulationTypeRaw[0] == 0 && modulationTypeRaw[1] == 0 )
{
    // "Other" ModulationType specified, use "Unmodulated" instead
   hlaObject.getRfModulationType().setUnmodulatedType();
}
else
{
    // Business as Usual
   hlaDecode( hlaObject.getRfModulationType(), rfModulationType, map );
}
```